src/DragAndDrop.jsx

```diff
@@ -248,16 +271,94 @@ const DragAndDrop = () => {
   const name = currentApp ? currentApp.name : '';
   const appType = currentApp ? currentApp.appType : '';
 
+  useEffect(() => {
+    if (mintComplete) {
+      setTimeout(() => {
+        setMintComplete(false);
+      }, 6000);
+    }
+  }, [mintComplete]);
+
+  useEffect(() => {
+    if (error) {
+      setTimeout(() => {

```

Comment:
- This seems arbitrary, and this kind of complex timing logic should happen in a popups/errors system component, not `DragAndDrop`.
- There is no cancellation so it will conflict with other errors
- This number should be a constant

Reply:
The popups/errors system component is <GenericLoadingMessage>.
But it is used different timecount(3000, 6000) for each GenericLoadingMessage>.
And fixed conflict with other erros as return clearTimeout